### PR TITLE
New package: FireDaemon.CertifyOne version 4.0.8

### LINKED ISSUE #154495: [Package Request]: FireDaemon Certify One
### How can we help?

I would like someone else to build the manifest.

### Please read and ensure the following

- [X] The installer meets the above requirements

### Please provide the following information

Download Page Url: https://www.firedaemon.com/download-certify-one
Publisher: FireDaemon Technologies Limited
Package Name: FireDaemon Certify One
Description: Verify the confidentiality, integrity, authenticity, and availability of encrypted communications.
Package Version: 4.0.8
Installer URL: https://download.firedaemon.com/FireDaemon-Certify-One/4.0.8/Certify-One-x64-4.0.8.exe

### PR DESCRIPTION
**Pull request by schembriaiden**
### Pull request has been created with [komac](https://github.com/russellbanks/Komac) v2.2.1 :rocket:

Resolves: https://github.com/microsoft/winget-pkgs/issues/154495
 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/154554)

### CODE REVIEW

**Comment by stephengillie** [comment]
Verify: 
image

File: manifests/f/FireDaemon/FireDaemonCertifyOne/4.0.8/FireDaemon.FireDaemonCertifyOne.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Created with komac v2.2.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: FireDaemon.FireDaemonCertifyOne
```
Comment (stephengillie) [inline manifests/f/FireDaemon/FireDaemonCertifyOne/4.0.8/FireDaemon.FireDaemonCertifyOne.installer.yaml:4]:
```suggestion
PackageIdentifier: FireDaemon.CertifyOne
```
Please update file name and path to match this change.

File: manifests/f/FireDaemon/FireDaemonCertifyOne/4.0.8/FireDaemon.FireDaemonCertifyOne.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# Created with komac v2.2.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+
+PackageIdentifier: FireDaemon.FireDaemonCertifyOne
```
Comment (stephengillie) [inline manifests/f/FireDaemon/FireDaemonCertifyOne/4.0.8/FireDaemon.FireDaemonCertifyOne.locale.en-US.yaml:4]:
```suggestion
PackageIdentifier: FireDaemon.CertifyOne
```
Please update file name and path to match this change.

File: manifests/f/FireDaemon/FireDaemonCertifyOne/4.0.8/FireDaemon.FireDaemonCertifyOne.yaml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# Created with komac v2.2.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.6.0.schema.json
+
+PackageIdentifier: FireDaemon.FireDaemonCertifyOne
```
Comment (stephengillie) [inline manifests/f/FireDaemon/FireDaemonCertifyOne/4.0.8/FireDaemon.FireDaemonCertifyOne.yaml:4]:
```suggestion
PackageIdentifier: FireDaemon.CertifyOne
```
Please update file name and path to match this change.

**Review by stephengillie — CHANGES_REQUESTED** [review]
Should this package be called just `FireDaemon.CertifyOne`?

**Comment by vikingnope** [comment]
@stephengillie, thank you for pointing that out, should be correct now :)

**Review by stephengillie — APPROVED** [review]

**Pull request merged**